service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ManageDocumentsController.java

```diff
@@ -153,7 +153,7 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(@RequestBod
                 break;
         }
 
-        removeTemporaryFields(caseDetails, TEMP_FURTHER_EVIDENCE_DOCUMENTS_COLLECTION_KEY, MANAGE_DOCUMENT_KEY,
```

Comment:
Why is this being removed?

Reply:
I don't know exactly now but there was some issue with ManageDocuments event and @Jamesnelson117 asked me to remove it. @Jamesnelson117 - Could you remind me what this is about please?

Reply:
@lewisbirks it's removal introduces a bug that was not caught originally. 

In about to submit
```
 case FURTHER_EVIDENCE_DOCUMENTS:
                List<Element<SupportingEvidenceBundle>> currentBundle = manageDocumentService
                    .setDateTimeUploadedOnSupportingEvidence(caseData.getFurtherEvidenceDocumentsTEMP(),
                        caseDataBefore.getFurtherEvidenceDocumentsTEMP()
                );
```

because we remove `TEMP_FURTHER_EVIDENCE_DOCUMENTS_COLLECTION_KEY` the temporary furtherEvidenceDocuments on caseDataBefore will always be null.

Reply:
@Jamesnelson117 - I thought you are doing this in your WIP ManageDocs PR?

Reply:
I'm doing similar as part of my PR, but if you don't have this fix of your PR then you also cannot assert that previous documents get updated.

Reply:
If it is being fixed in james' PR then I would leave current functionality in this one

Reply:
alright so from how it has been explained to me is that in the current master branch, all the temp docs will get updated each time because `caseDataBefore.getFurtherEvidenceDocumentsTEMP()` is null. So for an interim fix we can leave the case field in (not ideal) and then james can fix it properly in his PR. Am I understanding correctly?

Reply:
Exactly. My PR removes `TEMP_FURTHER_EVIDENCE_DOCUMENTS_COLLECTION_KEY` in favour for just the one caseField for all document types in manage documents but I will write a unit test to cover the above scenario.